pkg/storage/methodtests/insert_certificate_auth_test.go

```diff
@@ -117,7 +117,9 @@ func TestInsertCertificateAuth(t *testing.T) {
 
 		// then:
 		require.NoError(t, err)
-		require.EqualValues(t, expectedResult, certs)
+		require.Equal(t, expectedResult.TotalCertificates, wdk.PositiveIntegerOrZero(2))
```

Comment:
Maybe it's better tu use `assert` here, as it will allow to check everything at once if there wasn't an error, instead of failing on this line first, and then after fixing, failing on the next line.